src/ram/src/ram.tcl

```diff
@@ -45,8 +47,19 @@ proc generate_ram_netlist { args } {
     set read_ports $keys(-read_ports)
   }
 
+  set tapcell ""
+  set max_tap_dist 0
+  if { [info exists keys(-tapcell)] } {
+    if { [info exists keys(-max_tap_dist)] } {
+      set max_tap_dist $keys(-max_tap_dist)
+    } else {
+      utl::error RAM 21 "The -max_tap_dist argument must be specified with tapcell."
+    }
+    set tapcell $keys(-tapcell)
+  }
```

Comment:
Generate a warning if no tapcell is specified. Something like "No tapcell is specified. The generated layout may not pass Design Rule Checks."